### PR TITLE
Add subscriptions API ref

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

File: content/subscriptions.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+`id`                    | String   | The event's unique identifier.
+`target`                | String   | The URL to send the payload to.
+`event`                 | String   | The type of event to send notifications to.
+`status`                | String   | The status of the subscription (Can be either `active` or `inactive`).
+`createdAt`             | Datetime | The date when the subscription was created (ISO8601).
+`updatedAt`             | Datetime | The date when the subscription was updated (ISO8601).
+
+#### Available event types:
+
+LINK TO HELP ARTICLE HERE BEFORE MERGE
```
Comment (m1keil) [inline content/subscriptions.md]: *cough*

**Review by m1keil — CHANGES_REQUESTED** [review]

File: content/subscriptions.md (hunk 1 above)
Comment (nir0s) [reply content/subscriptions.md]: Oops. Should've been a draft.

**Comment by nir0s** [comment]
Converted to draft, as this requires that we actually release the API.

**Comment by m1keil** [comment]
dafaq is subscription?

**Comment by nir0s** [comment]
See https://app.intercom.com/a/apps/ynwy4f0x/articles/articles/5685041/show

It's part of the work done for the Zapier integration.

**Pull request merged**